Update limits.md (#9526)
cloudflare/cloudflare-docs: content/workers/platform/limits.md

Durable Objects scale well across Objects, but each object is inherently single-threaded. A baseline of 100 req/sec is a good floor estimate of the request rate an individual Object can handle, though this will vary with workload.

Durable Objects have been built such that the number of Objects in the system do not need to be limited. You can create and run as many separate objects as you want. The main limit to your usage of Durable Objects is the total storage limit per account - if you need more storage, contact your account team.

---

## Image Resizing with Workers 

> When using Image Resizing with Workers, refer to [Image Resizing documentation](/images/image-resizing/format-limitations/#limits-per-format) for more information on limits.

Reviewer: ```suggestion
When using Image Resizing with Workers, refer to [Image Resizing documentation](/images/image-resizing/format-limitations/#limits-per-format) for more information on the applied limits.
```

> ## Format limitations on Using Image Resizing with Workers 
> 
> It will be the same Limitations when using Image Resizing without workers. Please refer here: [Format limitations](/images/image-resizing/format-limitations/#limits-per-format).

Reviewer: ```suggestion
## Image Resizing with Workers 

When using Image Resizing with Workers, refer to [Image Resizing documentation](/images/image-resizing/format-limitations/#limits-per-format) for more information on limits.
```
